.dockerignore

```diff
@@ -0,0 +1,10 @@
+Dockerfile*
+docker-compose*
+.dockerignore
+.git
+.gitignore
+README.md
+LICENSE
```

Comment:
Do you need to sort this alphabetically?

Reply:
Not really